### PR TITLE
No more istances of marine law. (et similar policy)

### PR DESCRIPTION
**Pull request by Ghommie**
Replaces istances Marine Law with Space Law as per policy and rules, as well as adjust some strings of text to our scenario, fix a typo or two... and as well create more, which may be corrected throughout this PR.

Comments and suggestions deeply welcomed.

### CODE REVIEW

File: code/game/jobs/job/civilians.dm [hunk 1]
```diff
@@ -103,7 +103,7 @@ Make sure that the doctors and nurses are doing their jobs and keeping the marin
 	generate_entry_message(mob/living/carbon/human/H)
 		. = {"You are a civilian, and are not subject to follow military chain of command, but you do work for the USCM.
 You are tasked with keeping the marines healthy and strong, usually in the form of surgery.
-You are also an expert when it comes to medication and treatment. If you do not know what you are doing, adminhelp so a mentor can assist you."}
```
Comment (F-OS) [inline code/game/jobs/job/civilians.dm]: The verb is still adminhelp

File: code/game/jobs/job/civilians.dm [hunk 2]
```diff
@@ -150,7 +150,7 @@ You are also an expert when it comes to medication and treatment. If you do not
 	generate_entry_message(mob/living/carbon/human/H)
 		. = {"You are a civilian, and are not subject to follow military chain of command, but you do work for the USCM.
 You are tasked with researching and developing new medical treatments, helping your fellow doctors, and generally learning new things.
-Your role involves a lot of roleplaying, but you can perform the function of a regular doctor. Do not hand out things to marines without getting permission from your supervisor."}
+Your role involves some roleplaying and gimmickry, but you can perform the function of a regular doctor. You shouldn't hand out things to marines without getting permission from your supervisor."}
```
Comment (F-OS) [inline code/game/jobs/job/civilians.dm]: Let's be real here, do we really need to tell them not to hand out things? They're basically the chemists of TG.

File: code/game/jobs/job/command.dm [hunk 3]
```diff
@@ -41,12 +41,12 @@ var/datum/announcement/minor/captain_announcement = new(do_newscast = 1)
...
 	generate_entry_message()
-		. = {"Your job is HEAVY ROLE PLAY and requires you to stay IN CHARACTER at all times.
-While you support Weyland-Yutani, you report to the USCM High Command, not the corporate office.
+		. = {"As the commander of the USS Almayer you are held by higher standard and are expected to act competently.
+While you may support Weyland-Yutani, you report to the USCM High Command, not the corporate office.
 Your primary task is the safety of the ship and her crew, and ensuring the survival and success of the marines.
 Your first order of business should be briefing the marines on the mission they are about to undertake.
-If you require any help, use adminhelp to talk to game staff about what you're supposed to do.
-Godspeed, commander!"}
+If you require any help, use mentorhelp to talk about what you're supposed to do.
```
Comment (F-OS) [inline code/game/jobs/job/command.dm]: Ditto

**Comment by Ghommie** [comment]
Removed the changes on topic.dm in favour of [#352](https://github.com/ColonialMarines-Mirror/ColonialMarines-2018/pull/352).

**Comment by Rohesie** [comment]
This is good, but also due to change both on by https://github.com/ColonialMarines-Mirror/ColonialMarines-2018/pull/361 and when we make our own marine law soon (TM).

File: code/game/gamemodes/cm_initialize.dm [hunk 4]
```diff
@@ -402,11 +402,11 @@ datum/game_mode/proc/initialize_special_clamps()
 	if(is_queen)
 		to_chat(new_xeno, "<B>You are now the alien queen!</B>")
 		to_chat(new_xeno, "<B>Your job is to spread the hive.</B>")
-		to_chat(new_xeno, "Talk in Hivemind using <strong>:a</strong> (e.g. ':aMy life for the queen!')")
+		to_chat(new_xeno, "Talk in Hivemind using <strong>:a</strong> (e.g. ':a My life for the hive!')")
```
Comment (Rohesie) [inline code/game/gamemodes/cm_initialize.dm]: Changed by #361 to ;

File: code/game/jobs/job/civilians.dm [hunk 5]
```diff
@@ -179,10 +179,10 @@ Your role involves a lot of roleplaying, but you can perform the function of a r
 
 
 	generate_entry_message(mob/living/carbon/human/H)
-		. = {"As a representative of Weyland-Yutani Corporation, your job requires you to stay in character at all times.
+		. = {"As a representative of Weyland-Yutani Corporation you are expected to stay professional and loyal to the corporation at all times.
```
Comment (Rohesie) [inline code/game/jobs/job/civilians.dm]: Could you replace WY with NT?

Comment (Ghommie) [reply code/game/jobs/job/civilians.dm]: Yes, but on a separate PR. There are way too many istances in the game, plus a UI background, logos and icons. Doing this little change here would just confuse someone a little.

Comment (Rohesie) [reply code/game/jobs/job/civilians.dm]: Fair.

**Pull request merged**